troubleshooting/known_issues/ims_image_customization_job_status_stuck_at_waiting_on_user.md

```diff
@@ -0,0 +1,126 @@
+# IMS Image Customization Job Status Stuck at `waiting_on_user`
+
+## Issue description
+
+An IMS image customization job on a remote node can get stuck in the `waiting_on_user` state indefinitely.
+This can occur during `image customization` if any of the following things happen:
+
+- The remote node reboots or crashes
```

Comment:
```suggestion
An IMS image customization job on a remote build node can get stuck in the `waiting_on_user` state indefinitely.
This can occur during `image customization` if any of the following things happen:

- The remote build node reboots or crashes
```